regions/plotting/plotting.py

```diff
@@ -0,0 +1,58 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import SkyCoord
+from regions.io.read_ds9 import read_ds9
+from regions.shapes.circle import CircleSkyRegion
+import astropy.units as u
+
+
+def region_object_to_mpl_artist(region_list, ax=None):
+    """Convert a astropy region to a matplotlib artist"""
+    ids = {"<class 'regions.shapes.circle.CircleSkyRegion'>": 'skycircle',
+           "<class 'regions.shapes.circle.CirclePixelRegion'>": 'pixcircle',
+           "<class 'regions.shapes.ellipse.EllipseSkyRegion'>": 'skyellipse',
+           "<class 'regions.shapes.ellipse.EllipsePixelRegion'>": 'pixellipse',
+           "<class 'regions.shapes.polygon.PolygonSkyRegion'>": 'skypolygon',
+           "<class 'regions.shapes.polygon.PolygonPixelRegion'>": 'pixpolygon'}
+
+    output = list()
+
+    for reg in region_list:
+        temp = str(reg.__class__)
+        if temp in ids.keys():
+            region_type = ids[temp]
+            if region_type == 'skycircle':
+                patch = _skycircle_to_mpl_artist(reg, ax)
+        output.append(patch)
+
+    return output
+
+
+def _skycircle_to_mpl_artist(circle, ax, **kwargs):
+    
+    import matplotlib.patches as mpatches
+
+    val = circle.center.galactic
+    center = (val.l.value, val.b.value)
+    print(center)
+    temp = dict(transform=ax.get_transform('galactic'),
+                radius=circle.radius.to('deg').value)
+    kwargs.update(temp)
+    patch = mpatches.Circle(center, **kwargs)
+
+    return patch
+
+if __name__ == "__main__":
+    from astropy.io import fits
+    from astropy.wcs import WCS
+    import matplotlib.pyplot as plt
+
+    hdulist = fits.open('tests/empty_map.fits')
```

Comment:
instead of adding the empty_map to the repository, why not add the header and generate the empty_map data on the fly?